public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -102,7 +101,6 @@ export function AppChrome({ children }: Props) {
           <CommandPalette />
         </>
       )}
-      <TutorialProvider />
```

Comment:
You should be able to add this back without it annoying you now as I turned it off by default 😅

Reply:
Will do 👍